### DISCUSSION TITLE
milvus2.0千万数据集无法搜索

### DISCUSSION THREAD

**Discussion by GUSHUMING** [milvus-io/milvus#8286] [Q&A and General discussion]
以图搜图 milvus2.0 3千万数据集索引已经建立完毕但无法搜索出结果，Milvus Insight也非常卡，无法登录
![1632290730(1)](https://user-images.githubusercontent.com/73211918/134291384-a0fdb056-421d-4721-8758-44a24842413e.png)

在一百万数据集的时候搜索非常快，但在千万数据集上无法搜索。
索引为 default_index = {"index_type": "IVF_SQ8", "metric_type": METRIC_TYPE, "params": {"nlist": 16384}}
![image](https://user-images.githubusercontent.com/73211918/134291833-c94ff383-9096-415d-adc2-0aac046f6f0a.png)
请问这回是什么问题？是我资源分配不够吗？
下面是配置信息
version: '3.5'

services:
  etcd:
    container_name: milvus-etcd
    image: quay.io/coreos/etcd:v3.5.0
    volumes:
      - ${DOCKER_VOLUME_DIRECTORY:-.}/volumes/etcd:/etcd
    command: etcd -advertise-client-urls=http://127.0.0.1:2379 -listen-client-urls http://0.0.0.0:2379 --data-dir /etcd

  minio:
    container_name: milvus-minio
    image: minio/minio:RELEASE.2020-12-03T00-03-10Z
    environment:
      MINIO_ACCESS_KEY: minioadmin
      MINIO_SECRET_KEY: minioadmin
    volumes:
      - /ssd2/milvus/volumes/minio:/minio_data
    command: minio server /minio_data
    healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
      interval: 30s
      timeout: 20s
      retries: 3

  standalone:
    container_name: milvus-standalone
    image: milvusdb/milvus:v2.0.0-rc6-20210910-020f109
    command: ["milvus", "run", "standalone"]
    environment:
      ETCD_ENDPOINTS: etcd:2379
      MINIO_ADDRESS: minio:9000
    volumes:
      - ${DOCKER_VOLUME_DIRECTORY:-.}/volumes/milvus:/var/lib/milvus
    ports:
      - "19530:19530"
    depends_on:
      - "etcd"
      - "minio"

networks:
  default:
    name: milvus

**Comment by xiaofan-luan** [comment]
ehhh, can you offer logs of all nodes?
